### PR TITLE
add fc modules

### PR DESCRIPTION
**Pull request by muxiangqiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+            }
+            EOF
+
+         }
+
+2. Setting values for the following variables through environment variables:
+
+    - ALICLOUD_ACCESS_KEY
+    - ALICLOUD_SECRET_KEY
+
```
Comment (xiaozhu36) [inline README.md]: I think it also should set `ALICLOUD_REGION`

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+Usage
+-----
+You can use this in your terraform template with the following steps.
+
+1. Adding a module resource to your template, e.g. main.tf
+
+
+         module "tf-fc" {
+            source = "alibaba/fc/alicloud"
```
Comment (xiaozhu36) [inline README.md]: Line 26 should be `source = "terraform-alicloud-modules/alibaba/fc/alicloud"`

File: README.md (hunk 1 above)
Comment (muxiangqiu) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (muxiangqiu) [reply README.md]: done

**Pull request merged**